components/layout/SideNav.tsx

```diff
@@ -99,11 +100,14 @@ const Slide = styled.div`
   display: flex;
   overflow: hidden;
   width: 200%;
-  transform: translate3d(0, 0, 0);
-  transition: transform ease 0.4s;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  transition: left ease 0.4s;
 
   &.slidIn {
-    transform: translate3d(-50%, 0, 0);
+    left: -100%;
   }
```

Comment:
For whatever reason, Safari seems to have an issue using `transition` and `transform` together. I still don't know the root reason why, but it looks like this has been an issue for at least 5 years based on when this exact same issue was brought up in [this StackOverflow question](https://stackoverflow.com/questions/28288710/cant-scroll-inside-a-div-after-applying-webkit-transform-in-safari) 🤦

Reply:
Of course this fix only worked for desktop Safari and iOS is still broken 🤦 🤦